src/js/claims-status/utils/appeals-v2-helpers.jsx

```diff
@@ -1069,22 +1069,28 @@ export function getNextEvents(currentStatus, details) {
  * @returns {alertOutput} dynamically-generated title, description, and styling properties
  */
 export function getAlertContent(alert) {
-  const { type } = alert;
+  const { type, details, date, dueDate } = alert;
+  // TODO: confirm new date/dueDate usages
+  const formattedDate = moment(date, 'YYYY-MM-DD').format('MMMM DD, YYYY');
+  const formattedDueDate = moment(dueDate, 'YYYY-MM-DD').format('MMMM DD, YYYY');
+  // TODO: confirm status logic and property
+  const statusDescription = details.status === 'open' ? 'is active at the Board of Veterans’ Appeals' : 'is closed'; // Used in ramp_ineligible alert
```

Comment:
This is only used in the `ramp_ineligible` alert, should the variable name indicate this (e.g. `ramp_ineligibleStatusDescription`)? 
There are two texts available here: `is closed/is active at the Board of Veterans' Appeals`
Which statuses do these belong to (e.g. `open`, `denied`?) and how is the status accessed (e.g details.status)?

Reply:
This should use the `active` attribute. (line 62 here: https://app.swaggerhub.com/apis/dsva-appeals/appeals-status/2.0.0)

Reply:
That's right. `appeal.attributes.active`

Reply:
Thanks @cmgiven @cvalarida, will appeal.attributes.active be available on details?

Reply:
No, unfortunately not. You'll have to pass the `appeal.attributes.active` as a second parameter to this helper function from `src/js/claims-status/components/appeals-v2/AlertsList.jsx`. That's the only place this is called, so it should be pretty straightforward.

Reply:
Thanks @cvalarida, updated.